src/containers/library-item.jsx

```diff
@@ -21,6 +21,8 @@ class LibraryItem extends React.PureComponent {
             'startRotatingIcons',
             'stopRotatingIcons'
         ]);
+        // TODO: is there a better "is it an array?" check to use here?
+        this.hasIconsArray = props.icons.hasOwnProperty('length');
```

Comment:
```suggestion
        this.hasIconsArray = Array.isArray(props.icons);
```

Reply:
I'm afraid that's just a little too readable! ;)